frontend/views/containers/contributions/PaymentMethods.vue

```diff
@@ -1,66 +1,122 @@
 <template lang='pug'>
-fieldset
-  i18n.label(tag='legend') Payment methods
-  .c-list
-    label.c-item(
-      v-for='(method, index) in methodsAvailable'
+fieldset(data-test='paymentMethods')
+  legend.has-text-bold.c-legend
+    i18n.is-title-4 Payment info
+    | &nbsp;
+    i18n.has-text-1.has-text-small.c-optional (optional)
+  i18n.has-text-1 Other group members will be able to use this information to send you monthly contributions.
+
+  ul.c-fields(ref='fields' data-test='fields')
+    li.c-fields-item(
+      v-for='(method, index) in form.methods'
       :key='`method-${index}`'
+      data-test='method'
     )
-      .c-button(
-        type='button'
-        :disabled='!method.isAvailable'
-        :aria-pressed='selected === method.name'
-      )
-        .c-svg
-          component(:is='method.svg')
-        .radio.c-radio
-          input.input(type='radio' name='method'
-            @change='$emit("select", method.name)'
-            :disabled='!method.isAvailable'
-            :checked='method.name === selected'
+      .select-wrapper.is-reversed(:class='{"is-shifted": methodsCount > 1 }')
+        label
+          .sr-only Payment name
+          select.select(v-model='method.name'
+            :class='{ "is-empty": method.name === "choose"}'
+            @change='e => handleSelectChange(e, index)'
+          )
+            i18n(tag='option' value='choose' disabled='true') Choose...
+            option(v-for='(option, key) in config.options' :value='key') {{ option }}
+
+        label.c-select-input
+          .sr-only Payment description
+          input.input(
+            type='text'
+            v-model='method.value'
           )
-          span(:class='{ "has-text-1": selected !== method.name }') {{method.title}}
-        span.has-text-1.c-description {{method.description}}
+        button.is-icon-small.is-btn-shifted(
+          type='button'
+          :aria-label='L("Remove method")'
+          @click='removeMethod(index)'
+          data-test='remove'
+        )
+          i.icon-times
+
+  button.link.has-icon(
+    type='button'
+    @click='handleAddMethod'
+    data-test='addMethod'
+  )
+    i.icon-plus
+    i18n Add more
 </template>
 
 <script>
-import Tooltip from '@components/Tooltip.vue'
-import SvgBitcoin from '@svgs/bitcoin.svg'
-import SvgMoney from '@svgs/money.svg'
+import { mapGetters } from 'vuex'
+import Vue from 'vue'
 import L from '@view-utils/translations.js'
 
 export default {
-  name: 'PaymentsMethod',
-  components: {
-    SvgBitcoin,
-    SvgMoney,
-    Tooltip
-  },
-  props: {
-    selected: {
-      type: String,
-      required: true,
-      validator: (method) => ['manual', 'bitcoin'].includes(method)
+  name: 'PaymentMethods',
+  components: {},
+  data: () => ({
+    config: {
+      options: {
+        // key to store on state and corresponding translation (if needed)
+        bitcoin: 'Bitcoin',
+        paypal: 'Paypal',
+        venmo: 'Venmo',
```

Comment:
Do you know if other languages use different spellings for these? cc @dotmacro

Reply:
These are "trademarks" names and those usually don't suffer from translation, unless in some cases where they may adapt for different alphabets (ex: Chinese or Russian). Based on my experience with translations, I never needed to translate trademarks.